### PR TITLE
feat: create general purpose vscode operator function & finish select…

### PR DESCRIPTION
**Pull request by ayachensiyuan**
… azure tenant test case

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -368,6 +368,7 @@
     "@vscode/extension-telemetry": "^0.6.2",
     "ajv": "^8.12.0",
     "axios": "^1.6.0",
+    "dotenv": "^16.4.5",
```
Comment (wenytang-ms) [inline package.json:371]: just move this deps to devDeps

File: src/test/e2e/utils/constants.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import dotenv from "dotenv";
+dotenv.config();
```
Comment (wenytang-ms) [inline src/test/e2e/utils/constants.ts:5]: what's it used for?

Comment (ayachensiyuan) [reply src/test/e2e/utils/constants.ts:5]: changed

Comment (ayachensiyuan) [reply src/test/e2e/utils/constants.ts:5]: env for run on local machine

File: package.json (hunk 1 above)
Comment (ayachensiyuan) [reply package.json]: fixed

**Pull request closed without merging**